scripts/covid19indiaORG/Covid19IndiaORG.mcf

```diff
@@ -0,0 +1,7 @@
+Node: dcid:Count_MedicalConditionIncident_COVID_19_ConfirmedCase
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+statType: dcs:measuredValue
+measuredProperty: dcs:count
+medicalStatus: dcs:ConfirmedCase
+medicalCondition: dcs:COVID_19
```

Comment:
I am thinking we should stay consistent with https://datacommons.org/browser/CumulativeCount_MedicalConditionIncident_COVID_19_ConfirmedCase

and have incidentType instead of medicalCondition

Reply:
done